server/conf/evolutions/default/24.sql

```diff
@@ -0,0 +1,10 @@
+
+# --- !Ups
+ALTER TABLE block_rewards
+    ADD COLUMN staked_txid HASH_TYPE,
+    ADD COLUMN staked_index NON_NEGATIVE_INT_TYPE;
```

Comment:
I believe that If we store the amount, the queries could be simplified

Reply:
yeah, it would be one less join